internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/pending.go

```diff
@@ -95,7 +95,6 @@ func (h *Handler) AcceptReceivedShare(w http.ResponseWriter, r *http.Request) {
 			}
 		} else {
 			if s.State == collaboration.ShareState_SHARE_STATE_ACCEPTED {
-				s.Hidden = h.getReceivedShareHideFlagFromShareID(r.Context(), shareID)
```

Comment:
We do not need the hidden flag any more?

Do we still need `getReceivedShareHideFlagFromShareID`?

Reply:
I mean: Don't we need to set the flag somewhere else?

Reply:
there were two other places ... but both were actually unnecessary as well. we should have done a better review of the PRs ... 😞  https://github.com/cs3org/reva/pull/4454/commits/19b6b46043232bc0f0dd7b72c677fd1a45d35cfb